packages/runtime/container-runtime/src/runningSummarizer.ts

```diff
@@ -291,8 +294,8 @@ export class RunningSummarizer implements IDisposable {
                 const summarizeProps: ITelemetryProperties = {
                     summarizeReason,
                     summarizeTotalRetries: totalRetries,
-                    summarizeRetriesPerPhase: retriesPerPhase,
-                    summarizeAttemptPhase: attemptPhase,
+                    summarizeRetryPerPhase: retriesPerPhase,
```

Comment:
If we starting these numbers from 1, they should be called "attempts" not "retries" -> `attemptPhase`, `attemptsPerPhase` and `totalAttempts`. The very first summary attempt is not a retry so having them as 1 is not entirely correct.
If we want to call it "retries", they should start with 0 and should be increment after each attempt.

Reply:
renaming...